src/Component/Header/Header.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from "react";
+import "./style.scss";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Container } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import {
+  faSearch,
+  faUser,
+  faCartArrowDown,
+  faGlobe,
+  faBars,
+  faTimes,
+} from "@fortawesome/free-solid-svg-icons";
+function Header() {
+  const [toggleMenu, setToggleMenu] = useState(false);
+  const { t, i18n } = useTranslation();
+  const [language, setLanguage] = useState("en");
+
+  function handleChangeLanguage() {
+    if (language === "en") {
+      i18n.changeLanguage("vi");
+      setLanguage("vi");
+    } else {
+      i18n.changeLanguage("en");
+      setLanguage("en");
+    }
+  }
+
+  function handleToggleMenu() {
+    setToggleMenu((prevState) => !prevState);
+  }
+
+  return (
+    <header className="header">
+      <Container>
+        <div className="header_list">
+          <button className="header_btn-toggle " onClick={handleToggleMenu}>
```

Comment:
```suggestion
          <button className="header__toggle" onClick={handleToggleMenu}>
```
BEM này lạ nhỉ :v. Check lại các tên khác luôn nhé, 2 dấu chứ sao lại để 1 dấu.
Với lại -- là modified, còn toggle là element thì phải là __